guacamole/src/main/webapp/app/login/loginModule.js

```diff
@@ -23,4 +23,4 @@
 /**
  * The module for the login functionality.
  */
-angular.module('login', ['element']);
+angular.module('login', ['element', 'navigation']);
```

Comment:
(due to not being able to inject anything into the routing code, and thus needing a constant URL that will always redirect the user to their proper home)